lib/utils/network_utils.dart

```diff
@@ -0,0 +1,133 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:matrix/matrix.dart';
+import 'package:tawkie/config/themes.dart';
+
+import 'matrix_sdk_extensions/matrix_locals.dart';
+
+bool containsFacebook(List<String> participantsIds) {
+  return participantsIds.any((id) => id.contains('@messenger2'));
+}
+
+bool containsInstagram(List<String> participantsIds) {
+  return participantsIds.any((id) => id.contains('@instagram2_'));
+}
+
+bool containsWhatsApp(List<String> participantsIds) {
+  return participantsIds.any((id) => id.contains('@whatsapp'));
+}
+
+bool containsLinkedin(List<String> participantsIds) {
+  return participantsIds.any((id) => id.contains('@linkedin'));
+}
+
+bool containsDiscord(List<String> participantsIds) {
+  return participantsIds.any((id) => id.contains('@discord'));
+}
+
+bool containsSignal(List<String> participantsIds) {
+  return participantsIds.any((id) => id.contains('@signal'));
+}
+
+String removeFacebookTag(String displayname) {
+  if (displayname.contains('(FB)')) {
+    displayname = displayname.replaceAll('(FB)', ''); // Delete (FB)
+  }
+  return displayname;
+}
+
+String removeInstagramTag(String displayname) {
+  if (displayname.contains('(IG)')) {
+    displayname = displayname.replaceAll('(IG)', ''); // Delete (Instagram)
+  }
+  return displayname;
+}
+
+String removeWhatsAppTag(String displayname) {
+  if (displayname.contains('(WA)')) {
+    displayname = displayname.replaceAll('(WA)', ''); // Delete (WA)
+  }
+  return displayname;
+}
+
+String removeLinkedinTag(String displayname) {
```

Comment:
I'm not sure why, but the `(LinkedIn)` is not removed in the chat list. It it successfully removed in the chat header :+1: .

![image](https://github.com/Tawkie/tawkie-app/assets/23584745/190d27e4-02f8-4852-ad83-94d61ea0d7b3)

Reply:
I should add that it is also successfully removed on the `dev` branch : 
![image](https://github.com/Tawkie/tawkie-app/assets/23584745/3557ece1-10b6-4ec9-bc25-713c87d00904)